.gitignore

```diff
@@ -9,3 +9,6 @@ venv/
 *.pyc
 app/db.sqlite3
 db.sqlite3
+
+staticfiles/
```

Comment:
Consider removing the redundant entry for `db.sqlite3` if `app/db.sqlite3` is sufficient for your needs. Having both might be unnecessary unless you have multiple database files in different locations.